modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/portlet/action/GetTrafficSourcesMVCResourceCommand.java

```diff
@@ -160,53 +183,54 @@ private JSONArray _getTrafficSourcesJSONArray(
 		ResourceBundle resourceBundle) {
 
 		Map<String, String> helpMessageMap = HashMapBuilder.put(
+			"direct",
```

Comment:
Could we create a static map just for the keys and translate them once are used?

Reply:
For sure, if you think this change is important, send a pull request changing this piece of code to use the static map and I will adapt this pull later :).

I won't merge refactors of our current core with the new solution, since this could make it hard to merge this pull.

Reply:
the same here,  I was never proud of my first approach...